setup.cfg

```diff
@@ -1,3 +1,8 @@
 # Inside of setup.cfg
 [metadata]
-description-file = README.md
\ No newline at end of file
+description-file = README.md
+
+[flake8]
+ignore = E501, F841, C901 # line length, unused variables, overly complex functions
+exclude = .git,__pycache__,build,dist
+max-complexity = 10
```

Comment:
This more or less describes the checks that the Slider integration is performing on pull requests.